tests/instruments/keithley/test_keithleyDAQ6510.py

```diff
@@ -26,63 +26,49 @@
 from pymeasure.instruments.keithley import KeithleyDAQ6510
 
 
-def test_init():
-    with expected_protocol(
```

Comment:
You can keep the init test (albeit with an empty comm_pair_list), if you wish.

Reply:
Do you think it's a good idea? I'll add it back anyways.